compass/ocean/tests/drying_slope/viz/__init__.py

```diff
@@ -0,0 +1,50 @@
+import os
+import xarray
+import numpy
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from compass.step import Step
+from compass.ocean.tests.drying_slope.viz.plot import MoviePlotter, \
+    TimeSeriesPlotter
+
+
+class Viz(Step):
+    """
+    A step for visualizing drying slope results, as well as comparison with
+    analytical solution and ROMS results.
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='viz')
+
+        for damping_coeff in [0.0025, 0.01]:
+            self.add_input_file(filename='output_{}.nc'.format(damping_coeff),
+                                target='../forward_{}/output.nc'.format(
+                                       damping_coeff))
```

Comment:
Here is a rough idea of how the csv files would be added.  However, please don't hard-code these arrays as I have done here.
 
``` python
        for damping_coeff in [0.0025, 0.01]:
            for time in ['0.50', '0.05', '0.40', '0.15', '0.30', '0.25']:
                for datatype in ['analytical', 'roms']:
                    filename = f'r{damping_coeff}d{time}-{datatype}.csv'
                    self.add_input_file(filename=filename, target=filename,
                                        database='drying_slope')
```